src/abstractdataframe/show.jl

```diff
@@ -410,21 +404,27 @@ function showrows(io::IO,
     colwidths = maxwidths .+ textwidth(" ") .+ textwidth(" |")  # padding+border
     colwidths[end] += textwidth("|")  # add row column initial border
     coltextwidths = colwidths[end] .+ colwidths[1:end-1]
-    coloverflows = coltextwidths .> displaysize(io)[2]
-    chunkbounds = getchunkbounds(maxwidths, splitcols || !allcols, displaysize(io)[2])
 
-    nchunks = length(chunkbounds) - 1
-    nchunks = allcols ? nchunks : (chunkbounds[2] < 1 ? 0 : min(nchunks, 1))
-    nchunks = !allcols && splitcols && any(coloverflows) ? 0 : nchunks
+    chunkedcols = getchunkedcols(maxwidths, splitcols || !allcols, displaysize(io)[2])
+    chunkwidths = [colwidths[end] + sum(colwidths[cols]) for cols=chunkedcols]
```

Comment:
```suggestion
    chunkwidths = [colwidths[end] + sum(colwidths[cols]) for cols in chunkedcols]
```
seems more readable

Reply:
also why do we add `colwidths[end]` to every value?

Reply:
`colwidths[end]` represents the `Row` column width (see `getmaxwidths`, which eventually populates `maxwidths`). I updated the docstring in `showrows` to reflect that this is what is expected.

Reply:
Ah - OK, and in `chunkedcols` this last column is never included - right?

Reply:
That's right, in `getchunkedcols` (or `getchunkbounds` before I modified it), `maxwidths` doesn't influence the number of columns that get chunked.